sdk/schemaregistry/schema-registry-json/package.json

```
@@ -69,7 +69,7 @@
   },
   "dependencies": {
     "@azure/logger": "^1.0.0",
-    "@azure/schema-registry": "1.3.0-beta.2",
+    "@azure/schema-registry": "1.3.0",
```

```suggestion
    "@azure/schema-registry": "^1.3.0",
```

What do you think? Unless there's a reason not to that I'm just not aware of

---

That makes sense to me. I definitely forgot that, not any specific reason at all 😓